File: Perl/findEvenOrOddAndPrime.pl
# InComplete - | -

print "\n 1 for Odd or Even \n 2 for Prime or Not \n Enter the choice : ";
my $choice = <>;
if ($choice == 1){
    print "\n Enter the number : ";
    my $num = <>;
    if ($num%2 == 0){
            print " Even";
    }else{
            print " Odd";
    }
}
elsif (choice == 2){
    print "\n Enter the number : ";
    my $num = <>;
    if ($num > 1){
        for (my $i=2;$i<=($num/2);$i++){
            if($num%$i==0){
                print "not a prime";
                        break;
            }
        }else{
            print " prime";
        }
    }
    else{
        print " not a prime";
    }
}
else{
    print "\n Invalid choice , \n please enter correct choice :)"
}

File: python/findEvenOrOddAndPrime.py
choice = int(input('''
    1 for Odd or Even
    2 for Prime or Not

    Enter the choice : '''))

if choice == 1:
    num = int(input("Enter the number : "))
    if num % 2 == 0:
        print(num, " is Even")
    else:
        print(num, " is Odd")
elif choice == 2:
    num = int(input("Enter the number : "))
    if num > 1:

        for i in range(2, int(num/2)+1):
            if(num % i) == 0:
                print(num, " is not a prime")
                break
        else:
            print(num, " is a prime")

    else:
        print(num, " is not a prime")

else:
    print("Invalid choice , \n please enter correct choice :)")
